docs/subscription-plans.rst

```diff
@@ -62,6 +74,92 @@ Optimized**. Besides these, we also offer the CrateDB Cloud **Contract**.
   times the storage capacity.
 
 
+.. _subscription-plans-nodes:
+
+Subscription Plan Tiers and Nodes
+=================================
+
+Besides the hardware composition described above, which outlines the ratio of
+the elements CPU, RAM, and storage HD for each subscription plan, each plan has
+two further dimensions: tiers and nodes.
+
+Tiers are simply multiples of each hardware composition, and therefore allow
+linear scaling within a given plan. For example, the Basic tier of the General
+Purpose plan has 3.5 CPUs, 14 GiB of RAM, and 1 TiB of storage. The next tier
+of that plan, the Pro tier, has 7 CPUs, 28 GiB of RAM, and 2 TiB of storage,
+and so forth.
+
+The number of nodes in a cluster can also be adjusted manually for a given
+plan. Basic tiers of a plan allow scaling between 3, 4, and 5 nodes, whereas
+higher tiers allow scaling up to 15. (If you need more nodes than this, please
+`contact us`_.)
+
+The Development plan differs from the others: it has only a single tier and
+allows scaling between 1-3 nodes.
+
+.. WARNING::
+
+    When you have scaled a Development plan cluster above one node, you cannot
+    scale it back down to a single node again.
+
+Each node, in turn, corresponds to one DTU, regardless of tier. Put simply, the
+tier determines the hardware 'size' of the node, the number of nodes determines
+the number of DTUs at that size. Since the Development plan starts at 1 node,
+it starts at 1 DTU; other plans start at 3 nodes and therefore start at 3 DTUs.
+
+If you use our SaaS Marketplace offers, you will encounter DTUs/hour as the
+measure of actual usage that we bill for. If you subscribe to CrateDB Cloud
+directly via credit card, you do not have to concern yourself with DTUs. For a
+more detailed explanation of DTUs, see the section below.
+
+
+.. _subscription-plans-dtus:
+
+Explaining DTUs for scaling and billing
```

Comment:
I think we need to double-down and get rid of any public mentions of DTUs in the near future, as it would be much clearer just calling them node-hours :see_no_evil: